packages/theme/api-report.md

```diff
@@ -172,7 +178,42 @@ export function createBaseThemeOptions<PaletteOptions>(
   options: BaseThemeOptionsInput<PaletteOptions>,
 ): {
   palette: PaletteOptions;
-  typography: BackstageTypography;
+  typography:
+    | BackstageTypography
+    | {
```

Comment:
You'll want to look into why this return type broke into a union. Might suffice to explicitly put the type [on this line](https://github.com/backstage/backstage/blob/2fe23ff546814c963d51057fc9cb9574b3d485e8/packages/theme/src/base/createBaseThemeOptions.ts#L60), e.g. `const defaultTypography: BackstageTypography = {`